source/MaterialXRender/TinyObjLoader.cpp

```diff
@@ -195,7 +195,9 @@ bool TinyObjLoader::load(const FilePath& filePath, MeshList& meshList)
     mesh->setSphereCenter(sphereCenter);
     mesh->setSphereRadius((sphereCenter - boxMin).getMagnitude());
 
-    mesh->generateTangents(positionStream, texCoordStream, normalStream, tangentStream, nullptr);
+    MeshStreamPtr bitangentStream = MeshStream::create("i_" + MeshStream::BITANGENT_ATTRIBUTE, MeshStream::BITANGENT_ATTRIBUTE, 0);
```

Comment:
Need bitangent for testing. Just add storage since we always compute it anyways.